importer/importer.go

```diff
@@ -424,6 +431,18 @@ func (im *Importer) consumeMessages(pid int) {
 	defer im.wg.Done()
 	defer log.Println("Stopping worker", pid)
 	log.Println("Starting worker", pid)
+
+	conn, err := im.connect()
+	if err != nil {
+		panic(errors.Wrap(err, "unable to connect to clickhouse, discarding message"))
```

Comment:
Very small change: no need to write `discarding message` in this log, as we have not actually consumed any messages from the queue by this point.